lib/person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'unknown',
+                 parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age && @age >= 18
+  end
```

Comment:
- [ ] [OPTIONAL] Great job so far! You can simplify this code further by just using `age >= 18` directly. This comparison alone is sufficient to check if the age is greater than or equal to 18.